src/components/TabNavigation/TabNavigation.test.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { fireEvent, render, screen, within } from '@testing-library/react';
+import { TabNavigation } from './TabNavigation';
+import { Tab } from '../Tab/Tab';
+import userEvent from '@testing-library/user-event';
+
+describe('The TabNavigation component', () => {
+  render(
```

Comment:
```suggestion
  let tabNavigation: any;
  let tabs: any;

  beforeEach(() => {
    render(
      <TabNavigation>
        <Tab label="Live" articleCount={222}/>
        <Tab label="Scheduled" articleCount={333}/>
      </TabNavigation>
    );
    tabNavigation = screen.getByRole('tablist');
    tabs = within(tabNavigation).getAllByRole('tab');
  });
```

Reply:
Thank you @kkyeboah, I've changed that and also updated tests for LoginForm and AddStory where I was also rendering the same component before all tests are run - now the rendering code is in beforeEach().